.github/codejson/cookiecutter.json

```
@@ -1,67 +1,57 @@
 {
-    "project_name": "{{ cookiecutter.project_name }}", 
-    "project_repo_name": "{{ cookiecutter.project_repo_name }}",
-    "project_org": "{{ cookiecutter.project_org }}", 
+    "name": "my-project",
+    "project_name": "my-project",
```

I would replace the placeholder values with the cookiecutter variables from before (`{{ cookiecutter.project_name}}`, etc.) so it aligns better with the README.md instructions and errors out if the user forgets to add their variables.